pkgs/by-name/li/libmamba/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  cmake,
+  fmt,
+  spdlog,
+  tl-expected,
+  nlohmann_json,
+  yaml-cpp,
+  simdjson,
+  reproc,
+  libsolv-with-conda,
+  curl,
+  libarchive,
+  zstd,
+  bzip2,
+  python3,
+}:
+stdenv.mkDerivation rec {
+  pname = "libmamba";
+  version = "1.5.7";
+  src = fetchFromGitHub {
+    owner = "mamba-org";
+    repo = "mamba";
+    rev = "${pname}-${version}";
+    hash = "sha256-HfmvLi9IBWlaGAn2Ej4Bnm4b3l19jEXwNl5IUkdVxi0=";
+  };
+  nativeBuildInputs = [
+    cmake
+    python3
+  ];
+  propagatedBuildInputs = [
```

Comment:
Why propagated?

Reply:
I found myself repeating the same depencies that were needed for `libmambapy` and I understood `propagatedBuildInputs` to be the correct choice here.

Reply:
I don't agree with putting things in `outputs.dev`

So I just did the way I should have done it in the first place: I replaced `propagatedBuildInputs` with `buildInputs.
For `libmambapy`, I added the needed packages in `buildInputs` and noticed that there were a couple that I could remove.

They are ultimately different packages, so they could eventually evolve differently and I think this is the best approach long term.

Reply:
> I don't agree with putting things in outputs.dev

This is a fairly normal use for `propagatedBuildInputs` in Nixpkgs, but I'm happy you found a way to avoid them altogether

Reply:
Do you have an example?